src/adhocracy_mercator/adhocracy_mercator/evolution/__init__.py

```diff
@@ -55,14 +55,38 @@ def evolve2_disable_add_proposal_permission(root):  # pragma: no cover
                 (Deny, 'role:creator', 'add_mercator_proposal_version')]
     updated_acl = deny_acl + acl
     set_acl(root, updated_acl, registry=registry)
-    # TODO substanced bug, _p_changed is not set
-    root._p_changed = True
 
     logger.info('Finished substanced evolve step 2:'
                 'remove add_proposal permission')
 
 
+def evolve3_disable_voting_and_commenting(root):
+    """Disable rate and comment permissions."""
+    from adhocracy_core.utils import set_acl
+    from substanced.util import get_acl
+    from pyramid.threadlocal import get_current_registry
+    from pyramid.security import Deny
+
+    logger.info('Running substanced evolve step 3:'
+                'remove add_rate, add_rateversion, add_comment and'
+                'add_commentversion permissions')
```

Comment:
We repeat the numer and docstring here. That means if we change this we have to change it 
in 3 places. I propose to remove this comment completly or remove the number and reuse the docsting.